docs/design2.md

```diff
@@ -0,0 +1,110 @@
+# Key Transparency v0.5
```

Comment:
Maybe "Key Transparency Redesign" or "... New Design"? The version number might end up different, maybe even v1?
Update the file name as well?